components-compose/src/main/java/uk/gov/hmrc/components/compose/molecule/input/TextInputView.kt

```diff
@@ -63,14 +47,17 @@ object TextInputView {
         labelContentDescription: String? = null,
         hintText: String? = null,
         hintContentDescription: String? = null,
-        prefix: @Composable (() -> Unit)? = null,
+        prefix: @Composable() (() -> Unit)? = null,
         placeholderText: String? = null,
         errorText: String? = null,
         errorContentDescription: String? = null,
         characterCount: Int? = null,
         maxChars: Int? = null,
         singleLine: Boolean = false,
-        keyboardOptions: KeyboardOptions = KeyboardOptions.Default
+        keyboardOptions: KeyboardOptions = KeyboardOptions.Default,
+        visualTransformation: VisualTransformation = VisualTransformation.None,
```

Comment:
Do we want people to be able to change the visual transformation for the normal text input view?